From fc5d08e524dd318a78dcb0c0e87c28b1dac65925 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Mon, 4 Nov 2013 17:22:43 +0800
Subject: [PATCH 0525/1543] ENGR00286724-3 ARM: dts: imx6q: change to generic
 name for flexcan

commit 0838826014e816c24657c26c40435dc70d00a6f5 from
git://git.freescale.com/imx/linux-2.6-imx.git

As following the device tree naming rule, we change the device node
name to a more generic one and use phandle name specificly.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit 0b65c43167403b43bdc4c36b66ab592ce2d68c1a)
---
 arch/arm/boot/dts/imx6dl-hummingboard.dts |    2 +-
 arch/arm/boot/dts/imx6qdl.dtsi            |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-hummingboard.dts b/arch/arm/boot/dts/imx6dl-hummingboard.dts
index 5bfae54..0391aab 100644
--- a/arch/arm/boot/dts/imx6dl-hummingboard.dts
+++ b/arch/arm/boot/dts/imx6dl-hummingboard.dts
@@ -61,7 +61,7 @@
 	};
 };
 
-&can1 {
+&flexcan1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_hummingboard_flexcan1>;
 	status = "okay";
diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 1b26fcb..1a8c15e 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -408,7 +408,7 @@
 				clock-names = "ipg", "per";
 			};
 
-			can1: flexcan@02090000 {
+			flexcan1: can@02090000 {
 				compatible = "fsl,imx6q-flexcan";
 				reg = <0x02090000 0x4000>;
 				interrupts = <0 110 IRQ_TYPE_LEVEL_HIGH>;
@@ -418,7 +418,7 @@
 				status = "disabled";
 			};
 
-			can2: flexcan@02094000 {
+			flexcan2: can@02094000 {
 				compatible = "fsl,imx6q-flexcan";
 				reg = <0x02094000 0x4000>;
 				interrupts = <0 111 IRQ_TYPE_LEVEL_HIGH>;
-- 
1.7.5.4

